### PR TITLE
Add CDN WAF support.

### PR DESCRIPTION
**Pull request by fore5fire**
Swagger review PRs: https://github.com/Azure/azure-rest-api-specs/pull/7442, https://github.com/Azure/azure-rest-api-specs/pull/5885

### CODE REVIEW

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CDN.")]
 
 [assembly: AssemblyVersion("4.0.0.0")]
```
Comment (isra-fel) [inline sdk/cdn/Microsoft.Azure.Management.Cdn/src/Properties/AssemblyInfo.cs]: Please update `AssemblyVersion` as well since it's a major version update

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @lsmith130 , could you
- Add a link to the swagger spec review PR in the description
- Update api tag in `AzSdk.RP.props`
- Update tests

Thanks

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (lsmith130) [reply sdk/cdn/Microsoft.Azure.Management.Cdn/src/Properties/AssemblyInfo.cs]: Done

**Comment by lsmith130** [comment]
> Hi @lsmith130 , could you
> 
> * Add a link to the swagger spec review PR in the description
> * Update api tag in `AzSdk.RP.props`
> * Update tests
> 
> Thanks

The API tag in `AzSdk.RP.props` was already updated to the preview version in master in an earlier PR. I'll leave it as-is here and fix it to be the stable API version in master.

The other requested changes, adding the swagger spec review PR links, updating the Assembly version, and updating the tests have all been done.

**Review by isra-fel — APPROVED** [review]

**Comment by isra-fel** [comment]
@lsmith130 please don't sync up with master in a feature PR, makes it hard to review.

**Pull request merged**